test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -398,5 +399,163 @@ public void OnnxModelInMemoryImage()
                 foreach (var score in dataPoint.Scores)
                     Assert.True(score > 0);
         }
+
+        private class ZipMapInput
+        {
+            [ColumnName("input")]
+            [VectorType(3)]
+            public float[] Input { get; set; }
+        }
+
+        private class ZipMapStringOutput
+        {
+            [OnnxSequenceType(typeof(IDictionary<string, float>))]
```

Comment:
How about tests for the new `OnnxMap` type as well?

Reply:
Unfortunately, ONNXRuntime doesn't allow us to create Map value as input and the Map value is always used as input of ONNX operator... I can add some smoke tests for sure.

---
In reply to: [298374926](https://github.com/dotnet/machinelearning/pull/3881#discussion_r298374926) [](ancestors = 298374926)